spec/src/main/asciidoc/javase/bootstrap_se.asciidoc

```diff
@@ -1,61 +1,131 @@
+    public SeContainerInitializer addDecorators(Class<?>... decoratorClasses);
+    public SeContainerInitializer addAlternatives(Class<?>... alternativeClasses);
+    public SeContainerInitializer addAlternativeStereotypes(Class<? extends Annotation>... alternativeStereotypeClasses);
+    public SeContainerInitializer addProperty(String key, Object value);
+    public SeContainerInitializer setProperties(Map<String, Object> properties);
+    public SeContainerInitializer addBeans(Bean<?>... beans);
+    public SeContainerInitializer disableDiscovery();
+    public SeContainerInitializer setClassLoader(ClassLoader classLoader);
+    public SeContainer initialize();
+}
+----
+
+Unless specified differently each methods of `SeContainerInitializer` returns the current `SeContainerInitializer` object.
+
+* `newInstance()` static method returns an instance of the implementation of `SeContainerInitializer` discovered by java service-provider.
+Each call returns a new instance of `SeContainerInitializer`.
+* `addBeanClasses()` add classes to the the synthetic bean archive
+* `addPackages()` add packages content to the synthetic bean archive.
+Versions exist by providing classes in the wished package or `Package` objects themselves.
+Other version of the method allows recursive addition
+* `addAnnotatedTypes()` add the provided `AnnotatedTypes` to the synthetic bean archive.
+* `addExtensions()` add the provided extensions (class or instance) to the synthetic bean archive.
+* `addInterceptors()` add the provided interceptors to the synthetic bean archive.
+* `addDecorators()` add the provided decorators to the synthetic bean archive.
+* `addAlternatives()` add the provided alternatives classes to the synthetic bean archive.
+* `addAlternativeStereotypes()` add the provided stereotypes classes to the synthetic bean archive.
+* `addProperty()` add a configuration property for the container
+* `SetProperties()` set the Map of configuration properties for the container.
+replace previous properties.
+* `addBeans()` add the provided beans to the set of discovered beans.
+* `disableDiscovery()` deactivate automatic type scanning and discovery.
+All bean archive will be ignored except the implicit bean archive.
+* `setClassLoader()` change the default class loader for the container
+* `initialize()` bootstrap the container and returns a `SeContainer` object as defined in <<se_container>>.
+
+
+
+Each time `initialize()` method is called on the `SeContainerInitializer` it returns a new `SeContainer<Object>` instance.
+The _application context_ is started automatically by the container on start up.
+An implementation need not support calling `initialize()` multiple times on `SeContainerInitializer` without the previously returned `SeContainer` being shutdown.
+
+
+[[se_container]]
+
+=== `SeContainer` class
+
+The `javax.enterprise.inject.se.SeContainer` inherits the `javax.enterprise.inject.spi.CDI` class as defined in <<provider>>.
```

Comment:
On second thought, this is less obvious that it seemed: it could be confusing to have use call `SeContainer.current();` or `SeContainer.getCdiProvider()` and get an exception.
Perhaps we should consider overriding these static methods or make `SeContainer` an interface inheriting `Instance<Object>` and only add a `getBeanManager()` method to it. WDYT @mkouba ?